src/platform/silabs/SiWx917/wifi/ethernetif.cpp

```diff
@@ -40,8 +40,8 @@
 
 #ifndef SILABS_LOG
 extern "C" {
-void efr32Log(const char * aFormat, ...);
-#define SILABS_LOG(...) efr32Log(__VA_ARGS__);
+void silabsLog(const char * aFormat, ...);
```

Comment:
why this can't use `#include "silabs_utils.h"`

Reply:
I believe it's because if we disable the logs the code doesn't compile unless we re-define the Macro.... will investigate soon.